core/src/components/datetime/test/basic/datetime.e2e.ts

```diff
@@ -55,7 +55,7 @@ test.describe('datetime: selecting a day', () => {
   });
 
   test('should not highlight a day until one is selected, with default-buttons', async ({ page }) => {
-    await testHighlight(page, 'custom-datetime');
+    await testHighlight(page, 'inline-datetime-no-value');
```

Comment:
```suggestion
    await testHighlight(page, 'custom-datetime');
```
I'm assuming this was edited by mistake; this is meant to check a datetime with `show-default-buttons` set.

Reply:
It wasn't by mistake. I ended up passing a value to `custom-datetime` in order to solve the issue with the changing time that we saw via Zoom. Since these tests ([test 1 ](https://github.com/ionic-team/ionic-framework/blob/main/core/src/components/datetime/test/basic/datetime.e2e.ts#L57)and [test 2](https://github.com/ionic-team/ionic-framework/blob/main/core/src/components/datetime/test/basic/datetime.e2e.ts#L53)) required a datetime that had no value then I ended up switching. However, I'll be reverting this file and moving the test to modal as [suggested](https://github.com/ionic-team/ionic-framework/pull/27207#discussion_r1173892310).